src/activate.ts

```diff
@@ -27,9 +28,23 @@ async function Activate(): Promise<void> {
                 const servicesConfig = core.getInput('services-config', { required: true });
                 await licenseClient.ActivateLicenseWithConfig(servicesConfig);
             } else {
-                const username = core.getInput('username', { required: true }).trim();
-                const password = core.getInput('password', { required: true }).trim();
-                const serial = core.getInput('serial', { required: license.toLowerCase().startsWith('pro') });
+                const pro = license.toLowerCase().startsWith('pro');
+                let username = core.getInput('username', { required: pro }).trim();
+                let password = core.getInput('password', { required: pro }).trim();
+
+                if (!username && !password) {
```

Comment:
If someone has the username but forgets the password will it not throw an error on them?